### PR TITLE
first version cloud-service

### PR DESCRIPTION
**Pull request by szihai**

### CODE REVIEW

File: docs/developers/cloud-service.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+KubeVela will delegate the lifecycle management of cloud service to Crossplane.
+
+## Install Crossplane (This demo uses crossplane version 0.13)
+
+You will need a Kubernetes cluster ver> 1.16 (Minikube and Kind clusters are fine).
+Also you will need to have KubeVela installed on the cluster.
+To provision a cloud resource, you need to have the Access Key and Secret to your Alibaba cloud account.
+
+* Create crossplane namespace: `kubectl create ns crossplane-system`
+* Install crossplane helm chart: `helm install crossplane  charts/crossplane/ --namespace crossplane-system`
```
Comment (hongchaodeng) [inline docs/developers/cloud-service.md:22]: This exists under examples/kubecondemo/ . Maybe Ask users to `cd $project_dir/examples/kubecondemo/ ` at the beginning?

File: docs/developers/cloud-service.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  * Create provider secret: `kubectl create secret generic alibaba-creds --from-literal=accessKeyId=<change here> --from-literal=accessKeySecret=<change here> -n crossplane-system`
+  * Configure the provider: `kubectl apply -f ../../examples/kubecondemo/script/provider.yaml`
+* Configure infrastructure: `kubectl crossplane install configuration crossplane/getting-started-with-alibaba:v0.13`
+
+So far we have configured Crossplane on the cluster.
+
+## Import the database workload definition
+
+First, register the db workload definition:   
+`kubectl apply -f ../../examples/kubecondemo/script/def_db.yaml`
```
Comment (hongchaodeng) [inline docs/developers/cloud-service.md:36]: If we ask users to cd to that dir, we don't need this prefix too.

**Review by hongchaodeng — APPROVED** [review]
lgtm after nit

File: docs/developers/cloud-service.md (hunk 1 above)
Comment (szihai) [reply docs/developers/cloud-service.md:22]: I'm not sure which folder will stay alive longer.

**Pull request merged**